### ISSUE TITLE
Mix-up between 2018.1.9 and 2018.9.1

### ISSUE THREAD

**Issue by hartwork** [Alir3z4/html2text#196]
Hi!

I noticed that multiple places mis-announce the January release as "2018.__9.1__" rather than "2018.__1.9__":
* `ChangeLog.rst`
* The release notes at https://github.com/Alir3z4/html2text/releases/tag/2018.1.9
* PyPI: https://pypi.python.org/pypi/html2text/2018.9.1

Are you aware?

**Comment by Alir3z4** [comment]
Oh, that's embarrassing.
I'll fix now.

Thanks for reporting.

**Comment by hartwork** [comment]
For the record, there are two instances of 2018.__9.1__ left at:
* In the change log: https://github.com/Alir3z4/html2text/blob/master/ChangeLog.rst#201891
* On the release page: https://github.com/Alir3z4/html2text/releases/tag/2018.1.9

**Comment by marksweb** [comment]
Yeah this just confused me. I copied the version from the top of the change log, put it in my `requirements.txt` & then pip couldn't find the version. I thought it was unreleased, not on pypi, but it's just a typo.

**Comment by hartwork** [comment]
@Alir3z4 can you fix the remaining leftovers?

**Comment by marksweb** [comment]
FYI I submitted a PR for the change log correction. Obviously the release page can't be updated.

**Comment by hartwork** [comment]
> FYI I submitted a PR for the change log correction.

Cool! I don't see it at https://github.com/Alir3z4/html2text/pulls though?

> Obviously the release page can't be updated.

It can :smiley:

**Comment by marksweb** [comment]
> It can 😃

Oh, you can edit tags, nice. Never tried until just now, so didn't know!
